pkg/summary/summarizers.go

```diff
@@ -304,11 +304,13 @@ func checkErrors(data data.Object, conditions []Condition, summary Summary) Summ
 
 	for _, c := range conditions {
 		status, found := conditionMapping[c.Type()]
-		if !found {
+		reasonIsError := c.Reason() == "Error"
+
+		if !found && !reasonIsError {
 			continue
 		}
 
-		if status.Has(metav1.ConditionStatus(c.Status())) {
+		if status.Has(metav1.ConditionStatus(c.Status())) || reasonIsError {
```

Comment:
nit: I'd probably put the `reasonIsError` test first to short-circuit checking `status.Has` if we already know it's an error.
```suggestion
		if reasonIsError || status.Has(metav1.ConditionStatus(c.Status())) {
```